docs/status.md

```diff
@@ -1,20 +1,41 @@
+offer no guarantees regarding backward or forward compatibility, and you should
+expect a certain amount of breakage with each ensuing k0smotron release.
+
+All caveats aside, though, we are truly excited about what k0smotron has to
+offer, and look forward to seeing how it can transform the way you manage your
+Kubernetes deployments. We are confident that the open source community will
+take great interest in the project, too, and help us to smooth out the rough
+edges in the days, weeks, and months ahead.
 
-At this point we can not give much guarantees over backwards and forwards compatibility so expect things to break a bit with upcoming releases of k0smotron.
+## Cluster API
 
-We wanted to release k0smotron as early as possible to see whether there is in general interest of running k0s control planes as Kubernetes resources. We are excited about the potential that k0smotron brings to the table, and we look forward to seeing how it can transform the way you manage your Kubernetes deployments.
+One key focus of the k0smotron project is to get it to function as a Cluster
+API provider for both `ControlPlane` and worker `Bootstrap` providers. Once
+this goal is achieved, you will be able to use the Cluster API to provision the
+control plane (within the management cluster) and the worker nodes in your
+favourite infrastructure supporting cluster API.
 
+## Known limitations
 
-## Cluster API
+Areas in which k0smotron is currently limited include:
 
-One of the directions we're looking at is the have k0smotron working as a Cluster API provider for both `ControlPlane` and worker `Bootstrap` providers. What this means is that you could utilize Cluster API to provision both the control plane, within the management cluster, and worker nodes in your favourite infrastructure supporting cluster API.
+* Control Plane configurability
 
-## Known limitations
+    The configurability of the [k0s](https://docs.k0sproject.io/stable/) control plane is not yet fully enabled.
+
+* Control plane exposure
+
+    Currently k0smotron only supports `NodePort` and `LoadBalancer` services,
+    and often it is necessary to further configure created services to ease
+    their interraction with cloud provider implementations.
+
+* Updates prevalence
 
-Some of the areas we know have lot of shortcomings currently:
-- Control Plane configurability: As you know, k0s itself has lot of configurability, we plan to enable full configurability of the k0s control plane
-- Control plane exposing: Currently k0smotron only supports `NodePort` and `LoadBalancer` type services. While that is in itself quite ok quite often there's need to further configure e.g. annotations etc. on the created service to make them play nice with cloud provider implementations.
-- Updates: While k0smotron is able to update the cluster controlplane easily the update does not strecth into worker nodes.
-- 
+    Although k0smotron can easily update the cluster controlplane, such updates
```

Comment:
This is true for standalone mode

Reply:
Inserted reference to standalone mode into bullet.